output/eu_crude/analysis_simplified.py

```diff
@@ -0,0 +1,61 @@
+import sqlite3 as lite
+import sys
+import matplotlib as plt
+from pyne import nucname
+
+con = lite.connect('eu_crude.sqlite')
+
+with con:
+
+    cur = con.cursor()
+
+    #initialized snf inventory
+    snf_inventory=0
+    
+    # Get all sink agentIds 
+    # SimId / AgentId / Kind / Spec / Prototype / ParentID / Lifetime / EnterTime
+    sink_id = []
+    agent = cur.execute("select * from agententry where spec like '%sink%'")
+
+    for ag in agent:
+        sink_id.append(ag[1])
+
+    #empty array for resources
+    resources=[]
+
+    # fetch from resources table all the resources that are sent to sink. 
```

Comment:
“A long descriptive name is better than a short enigmatic name. A long descriptive name is better than a long descriptive comment.” 
― Robert C. Martin, Clean Code: A Handbook of Agile Software Craftsmanship